### PR TITLE
feat(contributing): add the definition of standard change and small change for PR

### PR DESCRIPTION
**Pull request by shmpwk**
## Description

Add the definition of standard change and small change for PR template.

The motivation is to ensure that the PR authors do not confuse which template to use. 
e.g. even minor feature or bug fix PR should be written in standard template

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: docs/contributing/pull-request-guidelines/index.md [hunk 1]
```diff
@@ -48,6 +48,19 @@ Use this workflow as a reference when you contribute to Autoware.
 
 - The unified style of descriptions by templates can make reviews efficient.
 
+#### Condition
+
+- There are two types of template. One is [small change template](https://github.com/autowarefoundation/autoware/blob/main/.github/PULL_REQUEST_TEMPLATE/small-change.md) and the other one is [standard change template](https://github.com/autowarefoundation/autoware/blob/main/.github/PULL_REQUEST_TEMPLATE/standard-change.md). Please select one based on the following condition.
```
Comment (kenji-miyake) [inline docs/contributing/pull-request-guidelines/index.md:53]: https://developers.google.com/style/tone#politeness-and-use-of-please

```suggestion
- There are two types of template. One is [small change template](https://github.com/autowarefoundation/autoware/blob/main/.github/PULL_REQUEST_TEMPLATE/small-change.md) and the other one is [standard change template](https://github.com/autowarefoundation/autoware/blob/main/.github/PULL_REQUEST_TEMPLATE/standard-change.md). Select one based on the following condition.
```

File: docs/contributing/pull-request-guidelines/index.md [hunk 2]
```diff
@@ -48,6 +48,20 @@ Use this workflow as a reference when you contribute to Autoware.
 
 - The unified style of descriptions by templates can make reviews efficient.
 
+#### Condition
```
[the remark this answers is not in the document]
Comment (shmpwk) [reply docs/contributing/pull-request-guidelines/index.md]: FIxed in https://github.com/autowarefoundation/autoware-documentation/pull/344/commits/be84dba071afb803cd6563c7f92e3e78bc9e62c8

File: docs/contributing/pull-request-guidelines/index.md [hunk 3]
```diff
@@ -50,6 +50,20 @@ Use this workflow as a reference when you contribute to Autoware.
...
+
+2. Small Change:
+
+- Complexity: A small change typically involves low complexity, such as updating documentation, making simple refactoring, or adjusting code style. These changes can usually be understood and reviewed quickly, often without extensive collaboration or deep knowledge of the entire codebase.
+- Impact: A small change has a limited or localized impact on the system. It might affect only a specific component or have minimal implications for the system's overall functionality or performance. Small changes generally do not include minor feature additions or minor bug fixes, as those should be treated as standard changes due to their impact. These small changes can generally be merged more quickly and with less rigorous testing.
+
+#### Steps to use an appropriate pull request template
```
Comment (kenji-miyake) [inline docs/contributing/pull-request-guidelines/index.md:65]: Please do not add another section at the same level.

```suggestion
##### Steps to use an appropriate pull request template
```

File: docs/contributing/pull-request-guidelines/index.md [hunk 4]
```diff
@@ -50,6 +50,20 @@ Use this workflow as a reference when you contribute to Autoware.
...
+
+2. Small Change:
+
+- Complexity: A small change typically involves low complexity, such as updating documentation, making simple refactoring, or adjusting code style. These changes can usually be understood and reviewed quickly, often without extensive collaboration or deep knowledge of the entire codebase.
+- Impact: A small change has a limited or localized impact on the system. It might affect only a specific component or have minimal implications for the system's overall functionality or performance. Small changes generally do not include minor feature additions or minor bug fixes, as those should be treated as standard changes due to their impact. These small changes can generally be merged more quickly and with less rigorous testing.
+
```
Comment (kenji-miyake) [inline docs/contributing/pull-request-guidelines/index.md:64]: How about asking ChatGPT?

image
image

File: docs/contributing/pull-request-guidelines/index.md (hunk 3 above)
Comment (shmpwk) [reply docs/contributing/pull-request-guidelines/index.md]: fixed

File: docs/contributing/pull-request-guidelines/index.md [hunk 5]
```diff
@@ -50,6 +50,30 @@ Use this workflow as a reference when you contribute to Autoware.
 
 #### Example
 
+- There are two types of template. One is [small change template](https://github.com/autowarefoundation/autoware/blob/main/.github/PULL_REQUEST_TEMPLATE/small-change.md) and the other one is [standard change template](https://github.com/autowarefoundation/autoware/blob/main/.github/PULL_REQUEST_TEMPLATE/standard-change.md). Select one based on the following condition.
```
Comment (kenji-miyake) [inline docs/contributing/pull-request-guidelines/index.md:53]: I think you can remove this part and combine it with the below part.

```suggestion
There are two types of template. Select one based on the following condition.
```

File: docs/contributing/pull-request-guidelines/index.md [hunk 6]
```diff
@@ -50,6 +50,30 @@ Use this workflow as a reference when you contribute to Autoware.
...
+
+1. Standard Change:
```
Comment (kenji-miyake) [inline docs/contributing/pull-request-guidelines/index.md:55]: Please use sentence case.

```suggestion
1. Standard change:
```

File: docs/contributing/pull-request-guidelines/index.md [hunk 7]
```diff
@@ -50,6 +50,30 @@ Use this workflow as a reference when you contribute to Autoware.
...
+
+- Complexity:
+  - Moderate to high, including new features, architectural changes, or addressing multiple issues.
+  - Requires deeper understanding of the codebase and collaboration.
+- Impact:
+  - Broader system impact, affecting multiple components, functionality, or performance.
+  - Includes minor feature additions and minor bug fixes.
+  - Requires thorough testing and evaluation before merging.
+
+2. Small Change:
```
Comment (kenji-miyake) [inline docs/contributing/pull-request-guidelines/index.md:65]: Same here.

```suggestion
2. Small change:
```

File: docs/contributing/pull-request-guidelines/index.md (hunk 5 above)
Comment (shmpwk) [reply docs/contributing/pull-request-guidelines/index.md]: fixed in [c5dd772](https://github.com/autowarefoundation/autoware-documentation/pull/344/commits/c5dd7721d30e50153005e026bcf93304cd542d6d)

File: docs/contributing/pull-request-guidelines/index.md [hunk 8]
```diff
@@ -50,6 +50,30 @@ Use this workflow as a reference when you contribute to Autoware.
 
 #### Example
 
+- There are two types of template. Select one based on the following condition.
+
+1. Standard change:
```
Comment (kenji-miyake) [inline docs/contributing/pull-request-guidelines/index.md:55]: Sorry, I meant to add a link here.
btw, using permanent link would be better.

```suggestion
1. [Standard change](https://github.com/autowarefoundation/autoware/blob/main/.github/PULL_REQUEST_TEMPLATE/small-change.md):
```

Comment (shmpwk) [reply docs/contributing/pull-request-guidelines/index.md]: Fixed in [643252d](https://github.com/autowarefoundation/autoware-documentation/pull/344/commits/643252d36493ee21f83eda610e31cd6de5b16ea0)

File: docs/contributing/pull-request-guidelines/index.md [hunk 9]
```diff
@@ -50,6 +50,30 @@ Use this workflow as a reference when you contribute to Autoware.
 
 #### Example
 
+- There are two types of template. Select one based on the following condition.
```
Comment (kenji-miyake) [inline docs/contributing/pull-request-guidelines/index.md:53]: [style]
- I think `-` is unnecessary here.
- template -> templates?

```suggestion
There are two types of templates. Select one based on the following condition.
```

Comment (shmpwk) [reply docs/contributing/pull-request-guidelines/index.md]: Thank you!

**Comment by shmpwk** [comment]
I think the description is more simpler by [43df7ba](https://github.com/autowarefoundation/autoware-documentation/pull/344/commits/43df7baba31e7456fb71d101152c47836bda0ace)

**Review by kenji-miyake — APPROVED** [review]

**Pull request merged**